code/game/gamemodes/clown_ops/clown_weapons.dm

```diff
@@ -40,14 +40,14 @@
 /obj/item/clothing/shoes/clown_shoes/banana_shoes/combat/Initialize()
 	. = ..()
 	var/datum/component/material_container/bananium = GetComponent(/datum/component/material_container)
-	bananium.insert_amount(max_recharge, MAT_BANANIUM)
+	bananium.insert_amount_mat(max_recharge, /datum/material/bananium)
```

Comment:
i think `insert_amount_mat` requires a /datum/material instance

Reply:
It doesn't now at least